app/controllers/games_controller.rb

```diff
@@ -47,7 +42,13 @@ def game
         @answers << Answer.new(answer_data)
       end
     end
+  end
+
 
+  def show_mcq_round
```

Comment:
method must be created in the routes and must be the same path as redirect to in round !!